src/io/summarize.jl

```diff
@@ -0,0 +1,86 @@
+"""
+summarize(model::MetabolicModel, flux_result::Dict{String, Float64}; 
```

Comment:
the name is a bit over-generalized, compared to how specific thing this function does.
`print_flux_summary` perhaps?

Minor considerations:
- as with all print* functions it should accept a file handle to print to
- this doesn't compose well -- the functionality isn't available to other parts of program, because if you want the output here, you'd need to re-read it from the file stream (ugly)